tests/testdata/ilimodels/CIAF_LADM/Catastro_COL_ES.ili

```diff
@@ -1046,7 +1046,7 @@ VERSION "2017-07-12 V2.1.6"  =
       Exactitud_Vertical : 0 .. 1000 [Catastro_COL_ES_V2_1_6.cm];
       Exactitud_Horizontal : MANDATORY 0 .. 1000 [Catastro_COL_ES_V2_1_6.cm];
       Confiabilidad : BOOLEAN;
-      Posicion_Interpolacion (EXTENDED) : Catastro_COL_ES_V2_1_6.COL_InterpolacionTipo;
+      Posicion_Interpolacion (EXTENDED) : MANDATORY Catastro_COL_ES_V2_1_6.COL_InterpolacionTipo;
```

Comment:
Hi @gacarrillor

With later ili2db versions the tests with the model CIAF_LADM are broken. The reason is, that this model works not properly.

```
Error: /home/dave/dev/opengisch/QgisModelBakerLibrary/tests/testdata/ilimodels/CIAF_LADM/Catastro_COL_ES.ili:1049:An attribute with cardinality {0..1} can not extend one whose cardinality is {1}. The cardinality must be more restrictive.
Error: /home/dave/dev/opengisch/QgisModelBakerLibrary/tests/testdata/ilimodels/CIAF_LADM/Catastro_COL_ES.ili:1068:An attribute with cardinality {0..1} can not extend one whose cardinality is {1}. The cardinality must be more restrictive.
Error: /home/dave/dev/opengisch/QgisModelBakerLibrary/tests/testdata/ilimodels/CIAF_LADM/Catastro_COL_ES.ili:1178:An attribute with cardinality {0..1} can not extend one whose cardinality is {1}. The cardinality must be more restrictive.
```

Concerning this library and the tests I felt free to change the model. I hope this is okay for you.

Reply:
Separated a little bit #50 here I still have the problem with the export of the geopackage.

Reply:
Hi @signedav,
I see, thanks for reporting it.
If you're fine with that, you could just remove those problematic tests and the problematic model. It's true, it's been a while since it was added and at the time the checker couldn't mark it as invalid.